source/mongoimport.txt

```diff
@@ -41,13 +41,16 @@ tool.
 Run ``mongoimport`` from the system command line, not the 
 :binary:`~bin.mongo` shell.
 
-.. seealso::
-
-   :binary:`~bin.mongoexport` which provides the corresponding
-   structured data export capability.
+:binary:`~bin.mongoexport` provides the corresponding
```

Comment:
[suggestion]

I realize this copy existed prior to this PR, but I think we should consider simplifying this statement.

Reply:
Sure I updated to a similar but simpler:

```
:binary:`~bin.mongoexport` provides data export capabilities.
```